src/main/java/be/cytomine/client/models/ProcessingServer.java

```diff
@@ -32,4 +36,9 @@ public String getDomainName() {
         return "processing_server";
     }
 
+    public void deleteProcessingServer(Long id) throws CytomineException {
```

Comment:
It's the old way to do the job. Now we use the `delete()` method from the `Model` parent class

Reply:
done